### ISSUE TITLE
Can we get a 0.18 release?

### ISSUE THREAD

**Issue by zsiddique** [celluloid/celluloid#762]
We are currently using the 0.18.0.pre-release to fix this issue, https://github.com/celluloid/celluloid/issues/696.  But it seems with the changes in Rubygems you can no longer use pre-release to satisfy dependencies unless you pass the --pre flag. This issue was brought up almost a year ago so I think it would be safe to cut a release now.

**Comment by tarcieri** [comment]
There are still some open issues I'd like to address, and I haven't had time to go through and put together a proper changelog since it's been so long since the last release.

I'm also pretty busy right now, so I'm not sure if I'll even be able to look at this for another couple weeks.

If you're interested in helping out by fixing some of the open issues, I can provide some more direction.

**Comment by zsiddique** [comment]
Do we have a filter/list for open issues on the 0.18.0? I see the project for 0.18.0 but.  IMHO if this project is stalling its best to address only bugs against 0.18pre and cut a release and the future can be figured out then.

**Comment by tarcieri** [comment]
I can tag the 0.18-specific regressions. There are a few of them.

**Comment by zsiddique** [comment]
So we are not blocked on fixing all the 0-18 regression I have created a PR for the fix in #696 to be applied against the 0-17-stable branch.  You think this is something you guys can check out quickly and take a look.  This would fix this issue for us: https://github.com/launchdarkly/ruby-client/issues/79

**Comment by tarcieri** [comment]
@zsiddique there's a 0.18.0.pre2 release out... mind helping test it?

**Comment by zsiddique** [comment]
We ended up refactoring our codebase and removing celluloid

**Comment by tarcieri** [comment]
Closing in favor of #789
